libs/reactive-core/src/main/java/no/nav/testnav/libs/reactivecore/router/InternalHandler.java

```diff
@@ -1,17 +1,41 @@
 package no.nav.testnav.libs.reactivecore.router;
 
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.env.Environment;
 import org.springframework.stereotype.Component;
 import org.springframework.web.reactive.function.BodyInserters;
 import org.springframework.web.reactive.function.server.ServerRequest;
 import org.springframework.web.reactive.function.server.ServerResponse;
 import reactor.core.publisher.Mono;
 
 @Component
+@SuppressWarnings("java:S1172")
+@RequiredArgsConstructor
 public class InternalHandler {
-    public Mono<ServerResponse> isAlive(ServerRequest request) {
-        return ServerResponse.ok().body(BodyInserters.fromValue("OK"));
+
+    private final Environment env;
+    private String html;
+
+    public Mono<ServerResponse> isAlive(ServerRequest ignored) {
+        return ServerResponse.ok().body(BodyInserters.fromValue(body()));
     }
-    public Mono<ServerResponse> isReady(ServerRequest request) {
-        return ServerResponse.ok().body(BodyInserters.fromValue("OK"));
+    public Mono<ServerResponse> isReady(ServerRequest ignored) {
+        return ServerResponse.ok().body(BodyInserters.fromValue(body()));
     }
+
+    private synchronized String body() {
+        if (html == null) {
```

Comment:
html er string og string kan også være "", en sjekk på `isBlank(html)` er mer robust.(org.apache.commons.lang3.StringUtils).

Reply:
Ut fra gjeldende dependencies har vi valget mellom `io.micrometer.common.util.StringUtils.isBlank` og `org.apache.logging.log4j.util.Strings.isBlank`. Litt heavy å introdusere en dependency på `org.apache.commons:commons-lang3` kun for dette.

Testen går også på om `html` er null, i synchronized, dvs. en one time for å avgjøre om `html` skal settes. Blir `html` satt til empty string så vil ikke det endre seg dersom synchronized block kjøres gjentatte ganger, da det vil være nødvendig med en redeploy (i praksis: endring av kode siden NAIS da endrer bruken av environment variable `NAIS_APP_IMAGE`).